src/liballoc/rc.rs

```diff
@@ -710,7 +710,7 @@ impl<T: ?Sized> Rc<T> {
         }
     }
 
-    /// Returns a mutable reference to the inner value,
+    /// Returns a mutable reference into the given `Rc`,
```

Comment:
Is the use of "inner value" wrong here? Seems like the `&mut T` points to the `T` inside the `RcBox`?

Reply:
No I guess inner value would also work here. Do you think that is better? There are a few occurrences of "into the given `Rc`".

Reply:
Don't feel strongly either way, was just wondering if there was something semantic about it. If you like the change you made, keep it. :)

Reply:
Thinking about this some more, I tried to use "inner value" only when the address where the value is stored generally doesn't matter. So I think I'll stick to "reference into the `Rc`" here.